package.json

```diff
@@ -75,7 +75,7 @@
 		"eslint-plugin-no-jquery": "^2.7.0",
 		"eslint-plugin-react": "^7.33.2",
 		"eslint-webpack-plugin": "^4.0.1",
-		"grunt": "^1.5.3",
+		"grunt": "^1.6.1",
```

Comment:
Revert changes. We can't make updates here.

Reply:
i changed it